configurator/configurator.go

```diff
@@ -0,0 +1,136 @@
+package configurator
+
```

Comment:
Okay, I'm still missing something - do we executed `go fmt ...` command on the whole codebase? Here it looks like we use _tabs_ by default, and it looks to me that existing code has spaces. Do we want to reformat it as a separate PR?

Reply:
Change it to execute only in my files?

Reply:
The command is totally fine - but I can see that formatting for some files in that review is still inconsistent. Have a look on `configurator.go` and `configuration.go` in comparison to e.g. `_test.go` files you've added - there is still different indentation in both.

Purpose of that command would be to unify formatting, so all files should be in line after executing that command. If I've missed something do not hesitate to correct me. :)